packages/good-design/src/apps/bridge/MicroBridge.tsx

```diff
@@ -79,8 +78,12 @@ export const MicroBridge = ({
   const targetChain = sourceChain === "fuse" ? "celo" : "fuse";
   const [toggleState, setToggleState] = useState<boolean>(false);
 
-  const balances = useBalanceHook() ?? {};
-  const { wei, gdValue } = balances[sourceChain] ?? {};
+  // query balances every 5 blocks, so balance is updated after bridging
+  const { G$: fuseBalance } = useG$Balance(5, 122);
```

Comment:
Causes bad performant responsive UI.
Can this not just be done after having done a tx?

Or at least set `refreshOrNever` from `never` > `everyBlock` around doing a tx to update balance.

Reply:
this is only every 25 seconds, why would it cause UI issues?

Reply:
@sirpy Because of the usedapp hooks which have always caused excessive re-renders 
that translates to the input fields getting very slow / wonky in user input responsiveness

Reply:
as I said, this causes a re-render only every 25 seconds. so I still doesnt see why rerenders would happen, I haven't seen any lag in the inputs